### ISSUE TITLE
F-Droid out of date

### ISSUE THREAD

**Issue by gaul** [jquesnelle/nds4droid#4]
Google Play has nds4droid version 47 while F-Droid has 46:

https://play.google.com/store/apps/details?id=com.opendoorstudios.ds4droid
https://f-droid.org/repository/browse/?fdid=com.opendoorstudios.ds4droid

**Comment by fervi** [comment]
@andrewgaul 
New version doesn't change anything (except German language). Nds4Droid is probably dead

**Comment by tangalbert919** [comment]
@fervi no it's not. We shall continue on the life of this emulator.

**Comment by fervi** [comment]
@tangalbert919 Yep, but it was dead in 2016 :P But you'll now works on this project, so it may works great in the future

**Comment by Nintendo365** [comment]
Hello! I know this is a VERY old game. But I suggest you fix it. Because old games can be Amazing! In ways.
